### ISSUE TITLE
[BUG] copy-in command is failing on pot 13

### ISSUE THREAD

**Issue by bretton** [pizzamig/pot#182]
**Describe the bug**
The `copy-in` command is failing on `pot 0.13.0`

**To Reproduce**
file: `flavor`
```
copy-in -s /usr/local/etc/pot/flavours/flavour.d/local -d /root/.pot_local
```
try build an image with a potbuilder script, fails with error
```
cp: /tmp/copy-in.V4IhQ7Iz: No such file or directory
jail: /bin/cp -a /tmp/copy-in.V4IhQ7Iz /root/.pot_local: failed
###>  Source /usr/local/etc/pot/flavours/flavour.d/local NOT copied because of an error
```

**Expected behavior**
The directory `local` and contents are copied in to /root/.pot_local

**System configuration - if possible**
 - `/usr/local/etc/pot/pot.conf`
```
POT_NETWORK=10.192.0.0/10
POT_NETMASK=255.192.0.0
POT_GATEWAY=10.192.0.1
POT_EXTIF=vtnet0
```
**Additional context**
This works fine on `pot 0.11.6`

**Comment by grembo** [comment]
Fixed in #176

**Comment by grembo** [comment]
See also #174
